tests/banners/where-does-my-donation-go.js

```diff
@@ -0,0 +1,15 @@
+import { bannerConfig } from '../../config/banners.js';
```

Comment:
This file should be called something more descriptive of the action, maybe `desktop-shows-use-of-funds.js`